src/clj/fluree/db/json_ld/iri.cljc

```diff
@@ -10,6 +11,46 @@
 
 #?(:clj (set! *warn-on-reflection* true))
 
+(defrecord IRI [^String s]
+  Object
+  (toString [_] s))
+
+(defn iri?
+  [x]
+  (instance? IRI x))
+
+(defn string->iri
+  [s]
+  (->IRI s))
+
+(defn unwrap
+  [x]
+  (if (iri? x)
+    (str x)
+    x))
+
+(defn serialize-iri ^String
+  [iri]
+  (str "\"" iri "\""))
```

Comment:
Do we actually use this to store IRIs on disk? If it's just for pretty printing maybe `display-iri` is a better function name, so nobody is tempted to use it for storage in the future.

Reply:
I could go either way. We don't store iris on disk right now, but I do think we might eventually want to store iris on disk since sids can be ambiguous. The same namespace code could point to different namespaces from different ledgers. I had used "display" as this function name before, but when I came back and implemented the data reader for iris, serialize made more sense because the data reader takes the output of this function and turns it back into the iri, so serialize fit better considering all the ways this function is used.